libs/clients/national-registry/v3/src/lib/nationalRegistryV3.service.ts

```diff
@@ -0,0 +1,100 @@
+import { Injectable } from '@nestjs/common'
+import {
+  EinstaklingarApi,
+  EinstaklingurDTOAllt,
+  EinstaklingurDTOFaeding,
+  EinstaklingurDTOForsjaItem,
+  EinstaklingurDTOHeimili,
+  EinstaklingurDTOHju,
+  EinstaklingurDTOItarAuka,
+  EinstaklingurDTOLogforeldrar,
+  EinstaklingurDTOLoghTengsl,
+  EinstaklingurDTONafnAllt,
+  EinstaklingurDTORikisfang,
+  EinstaklingurDTOTru,
+  GerviEinstaklingarApi,
+} from '../../gen/fetch'
+import { ExcludesFalse } from './utils'
+
+@Injectable()
+export class NationalRegistryV3ClientService {
+  constructor(
+    private individualApi: EinstaklingarApi,
+    private fakeApi: GerviEinstaklingarApi,
+  ) {}
+
+  getAddress = (nationalId: string): Promise<EinstaklingurDTOHeimili | null> =>
+    this.individualApi.midlunV02EinstaklingarNationalIdHeimilisfangGet({
+      nationalId,
+    })
+
+  async getAllDataIndividual(
+    nationalId: string,
+  ): Promise<EinstaklingurDTOAllt | null> {
+    const data = await this.individualApi.midlunV02EinstaklingarNationalIdGet({
+      nationalId,
+    })
+    return data ?? null
+  }
+
+  getFamily = (
+    nationalId: string,
+  ): Promise<EinstaklingurDTOLogforeldrar | null> =>
+    this.individualApi.midlunV02EinstaklingarNationalIdLogforeldrarGet({
+      nationalId,
+    })
+
+  async getCustodians(
+    nationalId: string,
+  ): Promise<Array<EinstaklingurDTOForsjaItem> | null> {
+    const child =
+      await this.individualApi.midlunV02EinstaklingarNationalIdForsjaGet({
+        nationalId,
+      })
+
+    return (
+      child?.forsjaradilar?.filter(Boolean as unknown as ExcludesFalse) ?? null
```

Comment:
you can use `isDefined` function in `libs/shared/utils/src/lib/isDefined.ts` method instead of this type assertion.

` child?.forsjaradilar?.filter(isDefined) ?? null`

Reply:
Ah nice, didn't know that